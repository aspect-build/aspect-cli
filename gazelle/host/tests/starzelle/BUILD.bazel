load("//gazelle:gazelle.bzl", "gazelle_generation_test")

# General starzelle unit tests. Each test has its own *.star plugins and only the host language.

[
    gazelle_generation_test(
        name = "%s_test" % t,
        dir = t,
        env = {
            "TEST_STARZELLE_PLUGINS": "%s/%s" % (
                package_name(),
                t,
            ),
        },
        gazelle_binary = "//gazelle/host:gazelle_host_binary",
    )
    for t in [t.replace("/WORKSPACE", "") for t in glob(["*/WORKSPACE"])]
]
