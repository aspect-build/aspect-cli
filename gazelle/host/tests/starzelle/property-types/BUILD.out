load("@test//:x.bzl", "props_test")

props_test(name = "no_defaults")

props_test(
    name = "with_defaults",
    b = True,
    n = 123,
    s = "default-value",
    sa = "1,2,3",
)
